common/src/main/kotlin/flank/common/Files.kt

```diff
@@ -13,6 +13,10 @@ val userHome: String by lazy {
     if (isWindows) System.getenv("HOMEPATH") else System.getProperty("user.home")
 }
 
+val appDataDirectory: String by lazy {
+    if (isWindows) System.getenv("APPDATA") else System.getProperty("user.home")
```

Comment:
I had a feeling something related to HOMEPATH was going to be the issue. I assume that HOMEPATH you dont have write permissions on the VM'S right?

Reply:
exactly. User directory is for users 🎉  Just one working day to figure this out

Reply:
and basically this is good practice on Windows to keep app data into app data folder instead of user one